scripts/buildah-bud.sh

```diff
@@ -53,6 +53,11 @@ if [[ "${WORKSPACES_DOCKERCONFIG_BOUND}" == "true" ]]; then
     fi
 fi
 
+ENTITLEMENT_VOLUME=""
+if [[ "${WORKSPACES_RHEL_ENTITLEMENT_BOUND}" == "true" ]]; then
+    ENTITLEMENT_VOLUME="--volume /tmp/entitlement:/etc/pki/entitlement"
```

Comment:
We should use `$WORKSPACES_RHEL_ENTITLEMENT_PATH` instead of hard coding `/tmp/entitlement` (if at some point we change the `mountPath`, we won't have to change this as well.